File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/direct/comment/uncomment/direct/Java/Go/s780275612_commented.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var intA int
	fmt.Scan(&intA)

	dbAncer := math.Pow(float64(intA), 3)
	intAncer := int(dbAncer)
	fmt.Println(intAncer)
} // End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/cot/Java/C/s780275612_commented.c
#include <stdio.h>
#include <math.h>

int main() {
    int intA;
    // Input reading
    scanf("%d", &intA);

    // Calculation
    double dbAncer = pow(intA, 3);
    int intAncer = (int)dbAncer;

    // Output
    printf("%d\n", intAncer);
    // End of Code
}
